packages/terra-heading/src/terra-dev-site/doc/example/colors.css

```diff
@@ -0,0 +1,16 @@
+/* stylelint-disable color-named */
+.crimson {
+  color: Crimson;
```

Comment:
Similar comment to above, avoid browser shorthand and either reference at themeable variable or document it.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2211/commits/35775cd63f5cf0ae778b8946046ff2e9a349eea6.